protocol/cors/accept-acah.feature

```diff
@@ -0,0 +1,16 @@
+# This only applies to pre-flight checks since that is where Access-Control-Allow-Headers is used
+Feature: Server should explicitly list Accept under Access-Control-Allow-Headers
+
+  Background: Set up test container
+    * def testContainer = rootTestContainer.createContainer()
+
+  Scenario: OPTIONS request returns Accept in Access-Control-Allow-Headers
+    Given url testContainer.url
+    And headers clients.alice.getAuthHeaders('OPTIONS', testContainer.url)
+    And header Origin = 'https://tester'
+    And header Access-Control-Request-Method = 'GET'
+    And header Access-Control-Request-Headers = 'X-CUSTOM, Content-Type'
```

Comment:
I was experimenting with the limits of the rules, wondering if all headers would be returned but I will change it since the server may not respond with `Content-Type` in the allowed list on a `GET` request since it is meaningless.
I may try with and without the `Accept` header in the request because, being pedantic, the requirement suggests the server should explicitly list `Accept` without reference to whether or not it was asked for.

Reply:
Aside: I just realised I got that from an example here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers.